benchmark/scalene.sh

```diff
@@ -0,0 +1,3 @@
+export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
+
+scalene benchmark/basic.py
```

Comment:
I found the option "--reduced-profile" to be useful (it only shows the lines that consume a lot of resources). Like for mprof.sh, this could be documented here.